.changeset/breezy-days-refuse.md

```diff
@@ -0,0 +1,8 @@
+---
+'@roadiehq/backstage-plugin-github-pull-requests': major
+'@roadiehq/backstage-plugin-github-insights': major
+---
+
+BREAKING: Needs SCM auth API to be configured in the application.
```

Comment:
Maybe, we could either provide a snippet here or give a file reference to where this is done in our app (or Backstage)? As much as this is standard now it might help someone.